Week5/Week5_Exercise2.js

```diff
@@ -0,0 +1,56 @@
+let taskList = [];
+
+function deleteItem(event) {
+    event.target.parentNode.remove();
+}
+
+function addTask(event) {
+    if (event.keyCode !== 13) {
+        return;
+    }
+    // Store the task text in a variable
+    let task = event.target.value;
+    console.log(task);
+    // Add the task to the list
+    // First create a list item
+    let taskListItem = document.createElement("li");
+    // Next create a checkbox
+    let checkBoxListItem = document.createElement("input");
+    checkBoxListItem.type = "checkbox";
+    // Create a label for the checkbox
+    let labelListItem = document.createElement("label");
+    labelListItem.textContent = task;
+    // Create a delete button
+    let deleteButtonListItem = document.createElement("button");
+    deleteButtonListItem.textContent = "Delete";
+    deleteButtonListItem.addEventListener("click", deleteItem);
+    // Add the items to the DOM 
+    taskListItem.appendChild(checkBoxListItem);
```

Comment:
This is going to create a new list item every time a key is pressed in `todo_input`?